src/patternfly.js

```diff
@@ -0,0 +1,17 @@
+/** PF Alert Component **/
+require('./pf-alert/pf-alert.component.js');
```

Comment:
I like the external file, but I was letting the components require the modules needed. Do we really need to include the template here, for example?